webui/src/docs/rest-api.jsx

```diff
@@ -139,6 +139,9 @@ module.exports = function() {
                 training site. Use this URL for testing.
             </li>
         </ul>
+        <p>Please make sure that you are not using production instances for
+            creating test records or testing the API in general.
```

Comment:
We should provide or at least recommend a standardized value for "site_function" when B2Share is in production. That way people can develop a standard client library and check for the value.